crypto/stateproof/committableSignatureSlot.go

```diff
@@ -58,7 +58,7 @@ func (sc committableSignatureSlotArray) Marshal(pos uint64) (crypto.Hashable, er
 }
 
 func buildCommittableSignature(sigCommit sigslotCommit) (*committableSignatureSlot, error) {
-	if sigCommit.Sig.Signature.Signature == nil {
+	if sigCommit.Sig.Signature == nil {
```

Comment:
the idea here is to check if this slot is empty. effectively we check the internal mss signature.
would it be safer to create a `merklesignature.Signature.isEmpty`?